qiskit/transpiler/passes/basis/basis_translator.py

```diff
@@ -98,11 +120,26 @@ def run(self, dag):
         )
 
         # Search for a path from source to target basis.
-
         search_start_time = time.time()
         basis_transforms = _basis_search(
             self._equiv_lib, source_basis, target_basis, _basis_heuristic
         )
+
+        extra_basis_transforms = {}
```

Comment:
Maybe `qarg_local_basis_transforms` or similar would be clearer?

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7227/commits/e6f90030350c3d6606eaa20b2a8637730eb17173